libhoney.go

```diff
@@ -235,6 +261,7 @@ type dynamicField struct {
 //
 // Make sure to call Close() to flush buffers.
 func Init(config Config) error {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
(Tests that expect a deterministic `rand` have an explicit `Seed` called inside them immediately before that expected behavior is verified.)